src/tests/core/order_manager_test.py

```diff
@@ -0,0 +1,44 @@
+import unittest
+from src.core.order_manager import OrderManager
+from src.tests.utils.fixtures.order_fixture import build_order
+
+
+class OrderManagerTestCase(unittest.TestCase):
+    def setUp(self):
+        self.order_manager = OrderManager()
```

Comment:
Inconsistencies when adding empty lines, try to make it more consistent

Reply:
done!